package.json

```diff
@@ -25,7 +25,7 @@
   "author": "",
   "license": "MIT",
   "devDependencies": {
-    "@types/react": "~18.0.14",
+    "@types/react": "~18.2.0",
```

Comment:
Upgraded to match our version in the RN SDK/StreemNext. By having the same version, this fixes some of the Typescript errors I was seeing (specifically with DebugStreemshots.tsx)